### ISSUE TITLE
Error path for asset's path and error while building.

### ISSUE THREAD

**Issue by erguotou520** [parcel-bundler/parcel#93]
Here is my [demo repo](https://github.com/erguotou520/simple-vue-parcel)  
There are two errors:  
1. The png file can't show in the page as it's `src` attribute is not right(missing prefix `/dist/`).  
2. Error occurred while building with command `parcel build index.html`:
```
parcel build index.html
🚨  /Users/erguotou/workspace/temp/simple-vue-parcel/index.js: Cannot read property 'start' of undefined
    at module.exports (/Users/erguotou/.nvm/versions/node/v8.9.2/lib/node_modules/parcel-bundler/node_modules/_babel-to-estree@0.0.2@babel-to-estree/babylon-to-espree/attachComments.js:56:42)
    at module.exports (/Users/erguotou/.nvm/versions/node/v8.9.2/lib/node_modules/parcel-bundler/node_modules/_babel-to-estree@0.0.2@babel-to-estree/babylon-to-espree/index.js:35:3)
    at exports.toEstree (/Users/erguotou/.nvm/versions/node/v8.9.2/lib/node_modules/parcel-bundler/node_modules/_babel-to-estree@0.0.2@babel-to-estree/index.js:9:3)
    at module.exports (/Users/erguotou/.nvm/versions/node/v8.9.2/lib/node_modules/parcel-bundler/src/transforms/uglify.js:10:39)
    at <anonymous>
```

**Comment by aantthony** [comment]
I'm having the same issue. You can do `parcel build index.html --no-minify` but it still won't work since it adds '/dist/' to every path in the outputted HTML, resulting in a 404.

**Comment by chee** [comment]
this is a duplicate of #8 

you can get rid of your `/dist/` by adding `--public-url='/'` to your build command

**Comment by erguotou520** [comment]
The second issue was fixed in `0.0.3` with arg `--no-minify`. However the first issue about image `src` still exist.

**Comment by davidnagli** [comment]
Fixed in `master`, should start working for everybody once we release all the new changes :)
